toolchain/arm-none-eabi/linux_aarch64/BUILD

```diff
@@ -65,7 +65,7 @@ cc_arm_none_eabi_config(
     name = "config",
     gcc_repo = compiler,
     gcc_version = "9.2.1",
-    host_system_name = "x86-darwin",
+    host_system_name = "linux_aarch64",
```

Comment:
We should also change the next line if possible

Reply:
didn't notice :) fixed.